src/test/ui/extern/extern-crate-path-to-missing.rs

```diff
@@ -0,0 +1,6 @@
+// If `extern crate` resolution fails, subsequent resolution errors should be deduplicated
+extern crate foo; //~ ERROR can't find crate for `foo`
+
+fn main() {
+    foo::bar();
```

Comment:
I think this can be merged into `extern-crate-multiple-missing.rs` with both `foo::something` and `bar::something` tried.